src/main/java/ama/WebSecurityConfiguration.java

```diff
@@ -31,7 +31,6 @@ public BCryptPasswordEncoder bCryptPasswordEncoder() {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
-                .csrf().disable()
```

Comment:
It wasnt that ur tests were useless. It was already being covered in the integration tests. So it became redundant